app/services/course/material/download_service.rb

```diff
@@ -0,0 +1,64 @@
+class Course::Material::DownloadService
```

Comment:
@lowjoel Can I keep this name ? I think in future we might use this service to only perform download ? or we might want to download as other formats.

And `DownloadService.download_and_zip` already indicates that the format is zip.